### ISSUE TITLE
Pd crashes when trying to open any ossia related files

### ISSUE THREAD

**Issue by JRSV** [ossia/libossia#830]
Hello,

I am having an issue in Pd, I have installed the ossia lib through "deken". I have configured the search path and loaded the lib on start up, (Welcome to ossia library in Pd window). The problem is that when I try to open any file or example from ossia library, pd just crashes. I tried with pd running through terminal with verbose mode all it spit out was "Pd: signal 6". I am very interested in running ossia but I cant get passed this issue.

I am using the latest Pd. 0.54 and I am running it on a debian 11 based system AVlinux-MXedition 21.

any help is appreciated.

thanks

**Comment by JRSV** [comment]
I have updated the ossia external downloading it from this github as the version in Deken is outdated, but now it does not even load Pd. it crashes on start up. Running pd in terminal gives me this
`terminate called after throwing an instance of 'std::runtime_error'
  what():  libavahi-client.so: not found. Functionality may be reduced.
Pd: signal 6
`

**Comment by evanmtp** [comment]
I'm having issues with this too (also running Pd 0.54, but on Windows 10). I'm able to load libossia v2.0.0-rc1-2297086, but adding any ossia object causes Pd to crash. Interestingly, I'm able to load and use ossia objects from the same release in [plugdata](https://plugdata.org/).

**Comment by jcelerier** [comment]
will check this ASAP, a new release is way overdue...

**Comment by jcelerier** [comment]
I checked and think I got a quick fix but which will cause a memory leak..sadly it seems that the newer Pd versions aren't compatible with the Cicmwrapper library we were using at all. Since it's not maintained I think there's not much other way than rewrite without cicmwrapper - @avilleret what's your take on this ?

**Comment by JRSV** [comment]
yes, to my knowledge, CICM  objects and libraries had issues, although I only found real problems with their GUI. I would encourage to get rid of CICMwrapper. anyway, I will keep an eye on the fix.
cheers
